data/settings.js

```diff
@@ -6152,6 +6152,322 @@ protocol !indexer-worker {
+		seealso: [ 'mail_path' ],
+		text: `
+Path to the INBOX mailbox. The path doesn't have to be absolute - it is
+relative to the [[setting,mail_path]].
+
+This is often used with mbox format where INBOX is in \`/var/mail/\` while the
+rest of the folders are under the user's home directory.
+
+This can also be used to specify a different INBOX path with Maildir:
+
+\`\`\`[dovecot.conf]
+ mail_driver = maildir
+ mail_path = ~/Maildir
+ mail_inbox_path = ~/Maildir/.INBOX
+\`\`\``
+	},
+
+	mail_index_path: {
+		tags: [ 'mail-location' ],
+		values: setting_types.STRING,
+		default: '<same as mail_path setting>',
+		seealso: [
+			'[[link,design_indexes_index_format]]',
+			'[[link,mail_location_index_files]]',
+		],
+		text: `
+Location of [[link,design_indexes_index_format]].`
+	},
+
+	mail_index_private_path: {
+		tags: [ 'mail-location' ],
+		values: setting_types.STRING,
+		seealso: [ '[[link,shared_mailboxes_public]]' ],
+		text: `
+The private index files are used with shared mailboxes to provide private
+(per-user) message flags.`
+	},
+
+	mail_cache_path: {
+		tags: [ 'mail-location' ],
+		values: setting_types.STRING,
+		default: '<same as mail_index_path setting',
+		text: `
+Place \`dovecot.index.cache\` files to this directory instead of among the
+other index files. This may be used as an optimization to split most index
+files to the fastest (smallest) storage while keeping cache files in a slightly
+slower (larger) storage.`
+	},
+
+	mail_control_path: {
+		tags: [ 'mail-location' ],
+		values: setting_types.STRING,
+		text: `
+Location for (mailbox-format specific) control files.`
+	},
+
+	mail_alt_path: {
+		tags: [ 'mail-location' ],
+		values: setting_types.STRING,
+		seealso: [ '[[link,dbox_alt_storage]]' ],
+		text: `
+Specifies the [[link,dbox_alt_storage]] path.`
+	},
+
+	mail_alt_check: {
+		tags: [ 'mail-location' ],
+		values: setting_types.BOOLEAN,
+		default: 'yes',
+		seealso: [ 'mail_alt_path' ],
+		text: `
+Whether to perform a sanity check and warn if [[setting,mail_alt_path]] changes
+from the last access. This can catch accidentally broken configurations before
+users start reporting missing mails. The downside to this check is some
+additional disk IO.`
+	},
+
+	mailbox_list_layout: {
+		tags: [ 'mail-location' ],
+		values: setting_types.ENUM,
+		values_enum: [ 'fs', 'index', 'Maildir++' ],
+		default: 'fs',
+		seealso: [
+			'[[link,dbox]]',
+			'[[link,maildir]]',
+			'[[link,mbox]]',
+			'[[plugin,obox]]',
+		],
+		text: `
+Directory layout to use.
+
+| Value | Description |
+| --- | --- |
+| \`Maildir++\` | The default used by [[link,maildir]]. |
+| \`fs\` | The default used by [[link,mbox]] and [[link,dbox]]. |
+| \`index\` | The default used by [[plugin,obox]]. Uses mailbox GUIDs as the directory names. The mapping between mailbox names and GUIDs exists in \`dovecot.list.index*\` files. |
+
+::: info
+This setting may be overridden by specific [[setting,mail_driver]] settings.
+:::`
```

Comment:
This sounds wrong. It's really the opposite. The mail_driver provides the default, and this setting can be used to override it.